backend/tests/test_hacknight.py

```diff
@@ -2,23 +2,23 @@
 
 
 def test_get_hacknights_when_logged_in(
-    client, access_token, add_hacknights
+    client, tokens, add_hacknights
 ):
     """Test get list of hacknights for logged in user."""
     rv = client.get(
         '/hacknights/',
-        headers={'Authorization': 'Bearer {}'.format(access_token)}
+        headers={'Authorization': 'Bearer {}'.format(tokens["access"])}
```

Comment:
If we're using Python 3.6+: Maybe f-string?

Reply:
I wanted to, but unfortunately we are on 3.5.2 right now. #187 will change python version to 3.8